### PR TITLE
Fix validate.js url

### PR DESCRIPTION
**Pull request by roboli**
Fix url in README.md and improve fxo custom validators.

### CODE REVIEW

File: lib/validate.js [hunk 1]
```diff
@@ -5,20 +5,20 @@ var validate = require('validate.js');
...
   }
 
   if(value && !value.valid) {
-    return 'is not a valid ' + type + ' input';
+    return options.message || 'is not a valid ' + type + ' input';
   }
 };
 
-validate.validators.fxoDatetime = function(value, key, options) {
+validate.validators.fxoDatetime = function(value, options, key) {
```
Comment (fiznool) [inline lib/validate.js]: Could you expand on why the args have been swapped around?

**Comment by fiznool** [comment]
@roboli thanks for the PR, I have one question - see above.

File: lib/validate.js (hunk 1 above)
Comment (roboli) [inline lib/validate.js]: Sure, according to the docs, http://validatejs.org/#custom-validator

**Comment by fiznool** [comment]
Thanks for this @roboli!

**Pull request merged**